CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.3.0 / 2024-12-12
+
+### Fixed
+* Fix a bug with sending metrics by @michael-richey in https://github.com/DataDog/datadog-sync-cli/pull/307
```

Comment:
```suggestion
* Fixed a bug with sending metrics by @michael-richey in https://github.com/DataDog/datadog-sync-cli/pull/307
```

Reply:
@michaelcretzman I just noticed that "fix" is consistent with everything else in this file. Should it be "fix" instead of "fixed"?

Reply:
@michael-richey typically in release notes the past tense is used. So `fixed` is recommended.